Testing/SystemTests/tests/framework/PelicanReductionTest.py

```diff
@@ -0,0 +1,36 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2018 ISIS Rutherford Appleton Laboratory UKRI,
```

Comment:
Can you bump this copyright year to 2021 whilst you add release notes? If it gets forgotten though I won't hold up the PR for something this minor

Reply:
Added release note and updated the license details.